File: app/src/main/java/com/example/jaycee/pomdpobjectsearch/JNIBridge.java
package com.example.jaycee.pomdpobjectsearch;

public class JNIBridge
{
    static
    {
        System.loadLibrary("JNI");
    }

    public static native boolean initSound();
    public static native boolean killSound();
    public static native void playSoundFFFF(float srcX, float[] list, float gain, float pitch);
    public static native void playSoundFF(float gain, float pitch);
}

File: app/src/main/kotlin/JNIBridge.kt
package com.example.jaycee.pomdpobjectsearch;

public class JNIBridge
{
    companion object
    {
        init
        {
            System.loadLibrary("JNI")
        }
    }

    external fun initSearch(target: Long, horizon: Long) : Boolean
    external fun getAction(state: Long) : Long
}
